packages/core/src/App/Components/Layout/Header/wallets/account-info-wallets.tsx

```diff
@@ -159,24 +165,24 @@ const AccountInfoWallets = observer(({ is_dialog_on, toggleDialog }: TAccountInf
                     onClick={active_account?.is_disabled ? undefined : () => toggleDialog()}
                 >
                     {is_mobile ? (
-                        <MobileInfoIcon currency={active_account?.currency} is_virtual={active_account?.is_virtual} />
+                        <MobileInfoIcons wallet_account={linked_wallet} />
                     ) : (
-                        <DesktopInfoIcons
-                            wallet_account={linked_wallet}
-                            active_account={active_account}
-                            show_badge={linked_wallet.is_malta_wallet || linked_wallet.is_virtual}
-                        />
+                        <DesktopInfoIcons wallet_account={linked_wallet} />
+                    )}
+                    <BalanceLabel
+                        balance={active_account?.balance}
+                        currency={active_account?.currency}
+                        is_virtual={active_account?.is_virtual}
+                        display_code={active_account?.currency_config?.display_code}
+                    />
```

Comment:
Couldn't the values of `active_account` be destructured?

Reply:
Yes, Updated.